### DISCUSSION TITLE
Login problems with Plex and Pi hole

### DISCUSSION THREAD

**Discussion by steveuk23** [tteck/Proxmox#1347] [Help Needed]
Hi sorry for the daft question.
But I've created a few of your things now from the scripts and they are running well.
I installed Plex and Pi hole on default settings and when I load up the GUI on Plex and sign in with my email I get a not authorised kind of message and I'm unable to back out or try again etc.
And similar with Pi hole I'm unable to login and get wrong password message and I've ran the password reset command but still same error.
I can't find anywhere what the login details are , should I have chosen advanced install on both of these ?
Thanks

**Comment by tteck** [comment]
If you're being prompted for credentials that you didn't input, it means you made an error. Delete the LXC and begin again, making sure to remain on the shell until the script completes.

**Reply by steveuk23** [reply]
Thanks I'll try again.
I have tried it twice now and it completes successfully and displays the IP address to go to.
Do you mean further messages appear in the shell once I'm on the GUI ?

**Reply by tteck** [reply]
No, if you get completed successfully then you're OK. For Plex, I have no idea what you're doing wrong, but for Pi-hole you need to set your password following the provided instructions.

**Reply by steveuk23** [reply]
Thanks I'll try again when I'm home.
Is there a way I could reset the lxc so it goes back to the default installation like deleting cache type of thing or is it just a case of deleting it all and adding again.

**Reply by tteck** [reply]
Remove and rerun the script.
